src/quality-menu-button.js

```diff
@@ -59,6 +59,7 @@ class QualityMenuButton extends MenuButton {
 
     this.el_.setAttribute('aria-label', this.localize('Quality Levels'));
     this.controlText('Quality Levels');
+    this.setIcon('cog');
```

Comment:
This would allow more CSS to be removed. 

```suggestion
    if (!player.options().experimentalSvgIcons) {
      this.el().$('.vjs-icon-placeholder').classList.add('vjs-icon-cog');
    }
    this.setIcon('cog');
```

We'll also need to change the minimum peerDependency of video.js to 8.5.0 to use `setIcon()`

Reply:
No harm in having an extra check, but setIcon() checks for this property in the `Component` class :)